Source/ACE.Server/Factories/LootGenerationFactory.cs

```diff
@@ -2266,6 +2266,27 @@ private static double GetWieldReqMeleeDMod(int wield)
             switch (wield)
             {
                 case 0:
+                    if (chance < 20)
```

Comment:
with the way these are written, the first one only has 19% chance, all of the middle ones have a 20% chance, and the last one has a 6% chance

Reply:
there isn't a a good reference on what the drop rate is, just want the top 3 to have a lower chance.

Reply:
what i'm saying is, with the way the code is written, it looks like the intention was for the first one to be 20%, and the last one to be 5%

the first and last entries have off-by-1 errors